src/libs/addEmojiToComposer/baseAddEmojiToComposer.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Should we rename this lib to `addEmojiToTextInput` to make it clearer that it could be reused elsewhere and isn't necessarily specific to `Composer`?

Reply:
Commented over there, but I think a better solution could be moving `setTextAndSelection` to the `RNTextInput`

Reply:
hm, it's handled differently for native VS web. Then we'd need to create a separate `RNTextInput` for each platform. I thought it be cleaner to have it in the `Composer/index` as that's already separated by platform.